src/books.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  # ...
+
+  attr_reader :rentals
+
+  def initialize(_title, _author)
+    # ...
+    @rentals = []
```

Comment:
- [ ] A method for adding rentals should be present. please find a link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship